examples/CustomPresentationExample.swift

```diff
@@ -99,6 +99,24 @@ final class VerticalSheetTransition: NSObject, Transition {
 
     CATransaction.commit()
   }
+  
+  //uncomment to make animation interactive
+  /*
+  var i = 0.01
+  func start(withInteractiveContext context: InteractiveTransitionContext) {
+    let timer = DispatchSource.makeTimerSource(queue: DispatchQueue.main)
+    timer.scheduleRepeating(deadline: .now(), interval: .milliseconds(10), leeway: .milliseconds(10))
```

Comment:
Remember: transition instances are reused for both the present and dismiss transitions, so any instance variables need to be re-initialized before the transition begins. In this case, we need to remember to reset i to 0.01 when we start the interactive transition.

If we don't reset the `i` value then the transition will immediately complete the dismissal transition.